pom.xml

```diff
@@ -279,6 +279,11 @@
         <artifactId>jackson-databind</artifactId>
         <version>${jackson.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.module</groupId>
```

Comment:
That library is needed for **ChainsConfigLoaderTest , BlockchainConfigUpdaterTest** to read all JSON data properly. In case full node run is works without it.
I moved that dependency into 'test' scope.